docs/source/technical_tutorials/working_with_r/gedi_access_r.ipynb

```diff
@@ -0,0 +1,906 @@
+{
```

Comment:
<p>I suggest the following instead:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">download_dir = file.path(getwd(), "data")
dir.create(download_dir, showWarnings = FALSE, recursive = TRUE)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>